masurca/4.0.9/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM ubuntu:focal as app
+
+ARG MASURCA_VER="4.0.9"
+ARG MINIMAP2_VER="2.24"
+ARG BWA_VER="0.7.17"
+
+LABEL base.image="ubuntu:focal"
+LABEL dockerfile.version="1"
+LABEL software="masurca"
+LABEL software.version=$MASURCA_VER
+LABEL description="Genome Assembly and Analysis"
+LABEL website="https://github.com/alekseyzimin/masurca"
+LABEL license="https://github.com/alekseyzimin/masurca/blob/master/LICENSE.txt"
+LABEL maintainer="Erin Young"
+LABEL maintainer.email="eriny@utah.gov"
+
+RUN apt-get update && apt-get install -y python \
```

Comment:
I have a small suggestion here to reduce the size of the docker image by about 10 MB. Not much space saved but uncompressed the docker image is about 2GB, so it might be worth it 🤷 All pending that it doesn't impact functionality.

```
$ docker images | grep masurca
erin/masurca  4.0.9-no-install-recommends             096380276130   11 seconds ago   2GB
erin/masurca  4.0.9                                   b0455a8f0a6e   25 minutes ago   2.01GB
```

You can achieve this by adding `--no-install-recommends` flag and additionally install `ca-certificates` since it's required when using `wget` and `curl`:
```Dockerfile
RUN apt-get update && apt-get install -y --no-install-recommends \
  ca-certificates \
  python \
  .....
```

When I did this locally, the test layer still ran successfully.

Up to you, I have no experience using masurca/polca/any of the masurca-included tools.